snmp/metadata.csv

```diff
@@ -510,6 +510,34 @@ snmp.cwoTfoStatsReservedConn,gauge,,,,"[CISCO-WAN-OPTIMIZATION-MIB] This object
 snmp.cwoTfoStatsResetConn,count,,,,"[CISCO-WAN-OPTIMIZATION-MIB] This object contains total number of optimized TCP connections, which are reset since TFO was started or its statistics were last reset.",0,snmp,,
 snmp.cwoTfoStatsTotalNormalClosedConn,count,,,,"[CISCO-WAN-OPTIMIZATION-MIB] This object contains total number of optimized TCP connections which were closed normally since TFO was started or its statistics were last reset.",0,snmp,,
 snmp.cwoTfoStatsTotalOptConn,count,,,,"[CISCO-WAN-OPTIMIZATION-MIB] This object contains total number of TCP connections optimized since TFO was started or its statistics were last reset.",0,snmp,,
+snmp.cyberpower.ePDU2BankStatusLoad,gauge,,,,"[CPS-MIB] Indicates the current draw, in tenths of Amps, of the load on the Rack PDU bank being queried.",0,snmp,,
```

Comment:
Does it matter that this description is formatted differently from the others? The others are just e.g. "The peak current..." or "The power consumption..." instead of "Indicates the peak current..." or "Indicates the power consumption..."

I guess it's written that way in the MIB as well, so maybe we don't care.

Reply:
Yeah IMO we can stick to the MIB description here